src/components/cover/coverPreloadLink.tsx

```diff
@@ -0,0 +1,42 @@
+import React from "react";
```

Comment:
If you agree that `CoverPreloadLink` is unnecessary then remember to remove this component.

Reply:
yep